### ISSUE TITLE
Most of the data seems to be skipping a day.

### ISSUE THREAD

**Issue by airguru** [jakobzhao/virus#22]
I am observing this problem in the update process for few days already. Most of the countries that I randomly select show this behaviour:
![image](https://user-images.githubusercontent.com/728780/76497818-043ad000-643c-11ea-9405-e71b374cc9e9.png)

The last value is correct, but the historical data seem to be shifted one day forward, overwriting the second-to-last value. Confirmed on multiple countries where I have independent access to time series.

**Comment by jakobzhao** [comment]
@xuf3 Thank you airguru! Could you help us figure out the problem?

**Comment by jakobzhao** [comment]
Thanks @airguru, I also cced to @xuf3, 

airguru, can you help us figure out a solution to this issue?

**Comment by airguru** [comment]
@jakobzhao Yes, I would like to contribute to this project! I just feel I need a bit more feedback :)
I still like this project as the most convenient visualisation, but I would like it to have more improvements.

Let me look at where the shift might be happening.

Would you consider downloading data from https://www.worldometers.info/coronavirus/ as that seems like the fastest updated source I have seen so far. I could implement the downloader, for all countries excluding China and US.

**Comment by jakobzhao** [comment]
Thank you! Yes, we have considered world meters, which has multiple benefits. But it is not our immediate needs, because we already had a crawler for countries.  We are thinking of making a crawler for U.S. cases (check [here](https://coronavirus.1point3acres.com/en)), but not succeed.

**Comment by sijukoiply** [comment]
What about scrapping data from https://topic.newsbreak.com/covid-19.html?s=i0 for US

**Comment by jakobzhao** [comment]
good suggestion @sijukoiply we will consider that!!thank you! @xuf3 @whichsteven @lolakang @joshuaji03

**Comment by NikolaosAndritsosHNMS** [comment]
Sorry to intervene, but I noticed that from the virus.csv file, the entire data row of 2020-03-16 are missing.

**Comment by jakobzhao** [comment]
@NikolaosAndritsosHNMS your observation is absolutely correct. we have just recovered that row.
